server/statistics/hot_peer_cache.go

```diff
@@ -519,12 +523,15 @@ func coldItem(newItem, oldItem *HotPeerStat) {
 	newItem.AntiCount = oldItem.AntiCount - 1
 	if newItem.AntiCount <= 0 {
 		newItem.needDelete = true
+	} else {
+		newItem.allowAdopt = true
```

Comment:
Is there a problem? such as `allowAdopt = false` for a new item, and then a heartbeat turns to `allowAdopt = true`, and then other stores can adopt from it, and the same problem occurs repeatedly.

Reply:
t1: peer in store1 adopt from store2, with anticount is 2 and is not allowed to adopt
t2. store1 heartbeat and cold, peer in store1 is with anticount is 1 and allowed to adopt 

So I think it doesn't occur repeatedly, because anticount decrease